#an array of submap indexes and tf

int32 trajectory_id
int32 submapIndex
geometry_msgs/Vector3 transform
geometry_msgs/Vector3 projected